src/core/lib/attribute-manager.js

```diff
@@ -318,6 +318,21 @@ export default class AttributeManager {
     return this;
   }
 
+  /**
+   * Returns object containing all accessors as keys
+   * @return {Object} - accessors object
+   */
+  getAccessors() {
+    const accessors = {};
+    for (const attributeName in this.attributes) {
+      const attribute = this.attributes[attributeName];
+      if (attribute.accessor) {
+        accessors[attribute.accessor] = true;
```

Comment:
`attribute.accessor` may be an array.

Reply:
> attribute.accessor may be an array.

Yep good catch. Using the `updateTriggers` member now, it already contains what I need (and handles this case).